kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
-# Run the text based installer to see dependency info in screenshots
+
+# By default run the text based installer to see dependency info in screenshots
+<% if ENV['KS_GUI'] == '1' then "" else "text" end %>
+
```

Comment:
Allowing $KS_GUI=1 to move back to text mode, as requested by @abellotti.

I spoke to Alberto, and the GUI mode was useful early on when initially creating the kickstart file. If you failed to set a required option, it will prompt you for the missing value, which is easier in GUI mode. Now that we have a working kickstart and making only incremental changes, I believe text is more useful because it gives you a log of action that you can scroll up. GUI mode only shows you the current package that is being installed.